### PR TITLE
[AuthenticationServices] Update bindings for Xcode 14 beta 1 and 2

### PR DESCRIPTION
**Pull request by tj-devel709**
Binding updates for AuthenticationServices for Xcode 14

### CODE REVIEW

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
Looks good! Just a few minor comments.

File: src/authenticationservices.cs [hunk 1]
```diff
@@ -252,9 +252,11 @@ interface ASPasswordCredential : NSCopying, NSSecureCoding, ASAuthorizationCrede
 	[BaseType (typeof (NSObject))]
 	[DisableDefaultCtor]
 	interface ASWebAuthenticationSession {
+		[TV (16,0)]
```
Comment (mandel-macaque) [inline src/authenticationservices.cs:255]: The type has NoTV already.

File: src/authenticationservices.cs [hunk 2]
```diff
@@ -252,9 +252,11 @@ interface ASPasswordCredential : NSCopying, NSSecureCoding, ASAuthorizationCrede
...
 		[Export ("initWithURL:callbackURLScheme:completionHandler:")]
 		NativeHandle Constructor (NSUrl url, [NullAllowed] string callbackUrlScheme, ASWebAuthenticationSessionCompletionHandler completionHandler);
 
+		[TV (16,0)]
```
Comment (mandel-macaque) [inline src/authenticationservices.cs:259]: Same.

File: src/authenticationservices.cs [hunk 3]
```diff
@@ -269,7 +271,7 @@ interface ASWebAuthenticationSession {
 		[Export ("prefersEphemeralWebBrowserSession")]
 		bool PrefersEphemeralWebBrowserSession { get; set; }
 
-		[Mac (10,15,4), iOS (13,4)]
+		[Mac (10,15,4), iOS (13,4), TV (16,0)]
```
Comment (mandel-macaque) [inline src/authenticationservices.cs:274]: Same.

**Review by mandel-macaque — CHANGES_REQUESTED** [review]

**Comment by tj-devel709** [comment]
I also am getting this build failure: `error BI1079: bgen: Do not know how to make a signature for Security.SecCertificate in parameter `certificate' from AuthenticationServices.ASAuthorizationProviderExtensionLoginManager.SaveCertificate`
that I don't believe I was getting before. I will continue to look into this

**Comment by tj-devel709** [comment]
@mandel-macaque this one should be updated to reflect requested changes now!

**Comment by tj-devel709** [comment]
Needs updating to Beta 4

**Comment by tj-devel709** [comment]
Beta 4 changes found here: https://github.com/xamarin/xamarin-macios/pull/15604

**Comment by dalexsoto** [comment]
I think everything is now updated and covered in Manuel's PR https://github.com/xamarin/xamarin-macios/pull/15604 so probably we can close this one to avoid clashes

**Pull request closed without merging**